examples/update-existing-zone-addresses/README.md

```diff
@@ -0,0 +1,9 @@
+# Zone example
+
+Example that creates a zone for context-based restrictions. This example uses the IBM Cloud Provider to automate the following infrastructure:
+
+- Creates 2 VPCs.
+- Creates 2 Public Gateways.
+- Creates 2 VPC Subnets.
+- Creates a CBR Zone for the VPC.
+- Updates an existing CBR Zone created above with a new addresses containing a new VPC and a `compliance` serviceRef.
```

Comment:
if it is `new addresses` -> `with new addresses`
if it is  `new address`  -> `with a new address`

`containing a new VPC` - assuming its a different VPC not the one used/created by this module. If thats the case, can we call it as `a VPC` instead of `new VPC` ?